wremnants/include/utils.h

```diff
@@ -16,6 +16,17 @@ namespace wrem {
 // pdg value
 constexpr double muon_mass = 0.1056583745;
 
+    template <typename T>
+    ROOT::VecOps::RVec<T> absVal(const ROOT::VecOps::RVec<T> & val) {
```

Comment:
yeah, I think the root implementation of abs does the right thing. This function is not used anywhere for now, although it would make the usage of abs for Rvec more explicit, since one could imagine that "abs" is actually std::abs, but in fact std::abs would not work, while the python abs does not necessarily work properly with float